app/core/controllers/cities.js

```diff
@@ -4,16 +4,28 @@ const m = attract('core/models');
 module.exports = {
   create: async (req, res, next) => {
     try {
-      const city = await m.city.create(req.body);
-      return res.render('cities', city);
+      await m.city.create(req.body);
+      return res.render('cities', {
+        section: 'Cities',
+        cities: await m.city.find({ 'meta.status': 'active' })
+      });
     } catch (error) {
       return next(error);
     }
   },
 
-  read: async (req, res) => {
+  read: async (req, res, next) => {
```

Comment:
The logic in this method points to it being a `DELETE`. ;)

Reply:
This was my first solution for DELETE logic that I forgot to remove after using $.ajax. 
So the form does not support the delete method, I implemented POST method with params, but it is not optimal.
Now I'm removing this solution.